api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -2814,4 +2814,40 @@ public void saveOrder_shouldSetAutoExpireDateForReviseOrderWithSimpleDosingInstr
 		
 		assertNotNull(revisedOrder.getAutoExpireDate());
 	}
```

Comment:
I believe there was a test that ensured the existing order gets discontinued, and i would assume after your change it would fail and then it passes once you specify the expected exception class and message